Gemfile

```diff
@@ -4,3 +4,5 @@ source 'https://rubygems.org'
 
 # See companies-house.gemspec
 gemspec
+
+gem "activesupport", require: false
```

Comment:
Can we accomplish this with `add_development_dependency` in the gemspec?

Reply:
Probably yes, will do that.

Reply:
Actually, I just removed this completely it's not needed.